src/types/onyx/Report.ts

```diff
@@ -39,9 +36,6 @@ type Report = {
     /** The email address of the report owner */
     ownerEmail?: string;
 
-    /** List of primarylogins of participants of the report */
-    participants?: string[];
```

Comment:
NAB. We should remove that key here as well:
- src/pages/home/sidebar/SidebarLinksData.js: https://github.com/Expensify/App/blob/00933e121f0952b7d1c2a3d577a04d2acc208de8/src/pages/home/sidebar/SidebarLinksData.js#L134
- src/types/onyx/OriginalMessage.ts: https://github.com/Expensify/App/blob/00933e121f0952b7d1c2a3d577a04d2acc208de8/src/types/onyx/OriginalMessage.ts#L16

Reply:
Good catches, thanks!